types/pubnub/index.d.ts

```diff
@@ -158,6 +158,12 @@ declare class Pubnub {
 
     publishFile(params: Pubnub.PublishFileParameters): Promise<Pubnub.PublishFileResponse>;
 
+    channel(params: string): Pubnub.Channel;
+    channelGroup(params: string): Pubnub.ChannelGroup;
+    channelMetadata(params: string): Pubnub.ChannelMetadataType;
+    userMetadata(params: string): Pubnub.UserMetadataType;
+    subscriptionSet(params: Pubnub.subscriptionSetParameters): Pubnub.SubscriptionSet;
```

Comment:
@mohitpubnub  I can't seem to find the documentation for these methods. I looked at the Pubnub Class in the source code https://github.com/pubnub/javascript/blob/master/src/core/pubnub-common.js and it doesn't look like these are defined there either. Its been a while since I've worked with Pubnub so let me know If I'm missing something.
Also instead of using argument name `params` for `string`, it might make sense to name the variable name like `channelName` or `channelGroupName`.

Reply:
That's true!  This PR is a bit in advance to be reviewed.
Due to some reasons docs are not prepared and also implementation PR merge is delayed.
And now, Since couple of more updates are being introduced, We will need few more updated in types.

So for that reason I'm closing this PR to avoid confusions to other reviewers as well.

Once SDK released, I'll create a fresh PR. (will address parameter naming feedback there.)

Thank you for review & feedback!! :)